src/other/Table.js

```diff
@@ -30,7 +30,7 @@
     Table.prototype.publish("fixedHeader", false, "boolean", "Enable or disable fixed table header",null,{tags:["Private"]});
     Table.prototype.publish("fixedColumn", false, "boolean", "Enable or disable fixed first column",null,{tags:["Private"]});
     
-    Table.prototype.publish("fixedSize", false, "boolean", "Fix Size to Min Width/Height");
+    Table.prototype.publish("fixedSize", true, "boolean", "Fix Size to Min Width/Height");
```

Comment:
OK, then in the MegaChart it needs to be set to true for the legend to be visible.